.github/workflows/manubot.yaml

```diff
@@ -0,0 +1,41 @@
+name: Manubot
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    name: Build Manuscript
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+      - name: Install Environment
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          activate-environment: manubot
+          environment-file: build/environment.yml
+          auto-activate-base: false
+          miniconda-version: 'latest'
+      - name: Build Manuscript
+        shell: bash --login {0}
+        run: bash build/build.sh
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: output
+          path: output
+      - name: Deploy
+        if: (
+          github.ref == 'refs/heads/master' &&
+          github.event_name == 'push' &&
+          job.status == 'success')
+        env:
+          MANUBOT_SSH_PRIVATE_KEY: ${{ secrets.MANUBOT_SSH_PRIVATE_KEY }}
```

Comment:
perhaps we can also test if `secrets.MANUBOT_SSH_PRIVATE_KEY` is set and if not, provide a better error message

Reply:
Yes, better error messages would be valuable.  Setting up CI has been a common source of errors.

Reply:
Okay I will save this for a subsequent PR. Possibly that also removes the legacy travis deploy key method of deployment.